lib/std/ascii.zig

```diff
@@ -415,8 +519,7 @@ pub fn orderIgnoreCase(lhs: []const u8, rhs: []const u8) std.math.Order {
     return std.math.order(lhs.len, rhs.len);
 }
 
-/// Returns true if lhs < rhs, false otherwise
-/// TODO rename "IgnoreCase" to "Insensitive" in this entire file.
+/// Returns whether `lhs` < `rhs`.
```

Comment:
"lexicographically" would be nice here, for clarity

Something like:
```suggestion
/// Returns true if `lhs` < `rhs` lexicographically, false otherwise.
```

Reply:
Sounds good!

But, I don't even understand why we need to have this function? If we have this, why don't we also have `greaterThanIgnoreCase`? I don't know but `orderIgnoreCase(lhs, rhs) == .lt` is really not a lot to type.

Reply:
I believe it's for convenience when a `lessThan` function is needed for generics (e.g. `std.sort`)